dataRetriever/resolvers/nilEpochProviderByNonce.go

```diff
@@ -0,0 +1,22 @@
+package resolvers
+
+import "github.com/ElrondNetwork/elrond-go/dataRetriever"
+
+// TODO : replace this with a real implementation
+
+type nilEpochProviderByNonce struct {
```

Comment:
this could be implemented. You have all the data in EpochTrigger.

Reply:
Left as a fake implementation for now as we can test other stuff is working before this